lib/tasks/ingest.rake

```diff
@@ -389,6 +389,44 @@ namespace :ingest do
     end
   end
 
+  desc "Import all NYC schools"
+  # task import_all_nyc_schools: :environment do
+  task import_all_nyc_schools: :environment do
+    csv_text = File.read('data/2016_-_2017_School_Locations.csv')
+    rows = CSV.parse(csv_text, headers: true)
+    rows.each do |row|
+      row_hash = row.to_hash
+      school = School.where(code: row_hash['ATS SYSTEM CODE'].strip).first_or_initialize
+      school.name = school.name || row_hash['LOCATION_NAME'].strip
+      school.address_line_1 = school.address_line_1 || row_hash['PRIMARY_ADDRESS_LINE_1'].strip
+      school.state = school.state || row_hash['STATE_CODE'].strip
+      if row_hash['Location 1'].present?
+        school.address_line_2 = school.address_line_2 || row_hash['Location 1'].strip
+        school.borough = school.borough || borough(row_hash['Location 1'].strip)
+        school.postal_code = school.postal_code || row_hash['Location 1'].strip[-5..-1]
+      end
+      school.phone_number = school.phone_number || row_hash['PRINCIPAL_PHONE_NUMBER'].strip
+      school.created_at = Time.zone.now if school.id.nil?
+      school.updated_at = Time.zone.now
+      school.save
+      puts school.name
+    end
+  end
+
+  def borough(borough_string)
+    if borough_string.include?('Manhattan')
+      'Manhattan'
+    elsif borough_string.include?('Queens')
+      'Queens'
+    elsif borough_string.include?('Brooklyn')
+      'Brooklyn'
+    elsif borough_string.include?('Statten')
+      'Statten Island'
+    elsif borough_string.include?('Bronx')
+      'Bronx'
+    end
+  end
+
   desc "Deactivate schools from authorized list"
   task :deactivate_schools, [] => :environment do |t, args|
     path = 'db/2015SchoolUpdate/dropped-schools.csv'
```

Comment:
Hardcoded path.  It looks like old code, so we don't need to fix in this PR, but could we put a TODO marker on it to remember to fix later?